lib/src/dart_formatter.dart

```diff
@@ -202,17 +202,15 @@ class DartFormatter {
       {required bool patterns}) {
     // Enable all features that are enabled by default in the current analyzer
     // version.
-    var featureSet = FeatureSet.fromEnableFlags2(
-      sdkLanguageVersion: Version(2, 19, 0),
-      flags: [
-        'inline-class',
-        'class-modifiers',
-        if (patterns) 'patterns',
-        'records',
-        'sealed-class',
-        'unnamed-libraries',
-      ],
-    );
+    FeatureSet featureSet;
+    if (patterns) {
```

Comment:
The patterns part really is special here because the whole point of this function is a hack to workaround patterns breaking syntactic compatibility in a couple of corner cases.